app/views/checks/_check_answers_table.html.slim

```diff
@@ -19,7 +19,8 @@
             - anchor = "section-#{table.index + 1}" if table.index
             = link_to t("checks.check_answers.change"),
                       check_step_path_from_step(table.screen.to_sym, params[:assessment_code], anchor:, begin_editing: true),
-                      class: "change-link govuk-!-margin-top-1"
+                      class: "change-link govuk-!-margin-top-1",
```

Comment:
I've never noticed before, but we could add this style to the CSS, instead of declaring here. 

```
.change-link {
  font-weight: bold;
  margin-top: 5px;
}
```